### PR TITLE
restructuring+contrib+intro

### LINKED ISSUE #3152: Clear Documentation - user and developer perspective
### What

two different views shall be provided a) stack developer b) application developer (user)

### Acceptance Criteria (DoD)

Documentation structure is clarified and available

### How

_No response_

### PR DESCRIPTION
**Pull request by anmittag**
- complete restructuring an adaptation of score repo according to folder structure
- Added Introduction and Contribution Guide to cover all the first getting started infos
- Basically no new and no deleted information, mostly shifted from a to b and renaming
- added document-needs and references

### CODE REVIEW

**Review by masc2023 — CHANGES_REQUESTED** [review]
https://eclipse-score.github.io/score/pr-3224/users_guide/index.html, somehow empty?

https://eclipse-score.github.io/score/pr-3224/contribute/index.html#what-rules-to-follow
Part of Software Development Plan?

https://eclipse-score.github.io/score/pr-3224/contribute/index.html#where-to-find-more-trainings
Links Guides, are that trainings? Some are checklist, etc.?

**Comment by anmittag** [comment]
@masc2023 :

User`Guide: Yes, Moved to contributor Guide. User = Using the middleware Contributor = Providing Code, Test, etc to the middleware

What rules to follow: needs from wp__sw_development_plan

Training: needs from wp__training_path

**Comment by masc2023** [comment]
> @masc2023 :
> 
> User`Guide: Yes, Moved to contributor Guide. User = Using the middleware Contributor = Providing Code, Test, etc to the middleware
> 
> What rules to follow: needs from wp__sw_development_plan
> 
> Training: needs from wp__training_path

Ok, then may we need to revise some realizes later, seems not correct.

**Review by masc2023 — DISMISSED** [review]
Rework of realizes needed after merge

**Comment by anmittag** [comment]
Then it was good to summarize 😎

File: docs/contribute/development/python/python_coding_guidelines.rst [hunk 1]
```diff
@@ -68,7 +70,7 @@ Testing
   implementation details. Overly fine-grained tests lead to brittle code and high maintenance.
 - **Boundary conditions:** - Edge cases often break things. Think about invalid input, large data
   sets, empty cases, and concurrency issues.
-- **Enesure Traceability** - Make sure to use the pytest decorator where applicable to enable traceabillity throughout. More infos on how to use it `example here <https://eclipse-score.github.io/docs-as-code/main/how-to/test_to_doc_links.html>`_ and `more through documentation here <https://eclipse-score.github.io/docs-as-code/main/internals/extensions/source_code_linker.html>`_
```
Comment (RolandJentschETAS) [inline docs/contribute/development/python/python_coding_guidelines.rst:71]: Spelling "Enesure"

File: docs/contribute/general/folder_structure_convention.rst [hunk 2]
```diff
@@ -60,14 +72,13 @@ The following shows the folder structure of the platform repository (ordered alp
...
     │   ├── stakeholder/               -> Stakeholder requirements [:need:`wp__requirements_stkh`]
-    │   ├── tool/                      -> Tool requirements (not covered in tool repositories)
-    │   │                                 [:need:`wp__requirements_proc_tool`]
     │   └── platform_assumptions/      -> Assumptions of use on platform level [:need:`wp__requirements_sw_platform_aou`]
     ├── safety/                        -> Safety documentation on platform level (SEooC):
     │                                     [:need:`wp__platform_dfa`], [:need:`wp__platform_safety_manual`],
     │                                     [:need:`wp__platform_safety_package`], [:need:`wp__fdr_reports`],
     │                                     [:need:`wp__audit_report`]
     ├── score_tools/                   -> Tool list and [:need:`wp__tool_verification_report`]
+    │   └── tools_requirementes/       -> Tool requirements [:need:`wp__requirements_proc_tool`]
```
Comment (RolandJentschETAS) [inline docs/contribute/general/folder_structure_convention.rst]: tools_requirementes -> tools_requirements

File: docs/index.rst [hunk 3]
```diff
@@ -234,35 +240,36 @@ Infrastructure and Tooling
...
       Get to know the S-CORE tool stack, testing and CI infrastructure.
       Read the
       `infra & tooling docs <https://eclipse-score.github.io/infrastructure/dev/index.html>`_.
 
+   .. grid-item-card::
+
+      :octicon:`tools;1.5em`
+
+      Toolchains
+      ^^^
```
Comment (RolandJentschETAS) [inline docs/index.rst:256]: Is this underline to short ?

**Review by RolandJentschETAS — DISMISSED** [review]

File: docs/index.rst (hunk 3 above)
Comment (anmittag) [reply docs/index.rst:256]: No this is fine for the grid cards

File: docs/contribute/development/python/python_coding_guidelines.rst (hunk 1 above)
Comment (anmittag) [reply docs/contribute/development/python/python_coding_guidelines.rst:71]: fixed in commit ab2e48be65ba586360ae249873195ddfe3e15615

File: docs/contribute/general/folder_structure_convention.rst (hunk 2 above)
Comment (anmittag) [reply docs/contribute/general/folder_structure_convention.rst]: fixed in commit ab2e48be65ba586360ae249873195ddfe3e15615

**Review by RolandJentschETAS — DISMISSED** [review]

File: docs/safety/fdr_reports_safety_analyses_DFA.rst [hunk 4]
```diff
@@ -56,7 +56,7 @@ Please note that it is mandatory to fill in the "passed" column with "yes" or "n
         -
       * - REQ_01_02
         - Are the templates for Safety Analysis used?
-        - See :need:`doc__score_platform_dfa`
+        - See :need:`doc__platform_dfa`
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/safety/fdr_reports_safety_analyses_DFA.rst:59]: this need is coming from process_description

File: docs/quality/qms_report.rst [hunk 5]
```diff
@@ -314,24 +318,24 @@ Quality Management Generic work products
       - -
 
     * - :need:`wp__platform_dfa`
-      - :need:`doc__score_platform_dfa`
-      - :ndf:`copy('status', need_id='doc__score_platform_dfa')`
+      - :need:`doc__platform_dfa`
+      - :ndf:`copy('status', need_id='doc__platform_dfa')`
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/quality/qms_report.rst:322]: this need is comming for process_description

**Review by antonkri — DISMISSED** [review]

File: docs/quality/qms_report.rst [hunk 6]
```diff
@@ -314,24 +318,24 @@ Quality Management Generic work products
...
 
     * - :need:`wp__platform_handbook`
       - :need:`doc__platform_handbook`
       - :ndf:`copy('status', need_id='doc__platform_handbook')`
 
     * - :need:`wp__fdr_reports`
-      - :need:`doc__score_platform_safety_analysis_fdr`
-      - :ndf:`copy('status', need_id='doc__score_platform_safety_analysis_fdr')`
+      - :need:`doc__platform_safety_analysis_fdr`
+      - :ndf:`copy('status', need_id='doc__platform_safety_analysis_fdr')`
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/quality/qms_report.rst:330]: this need is comming for process_description

File: docs/quality/qms_report.rst [hunk 7]
```diff
@@ -314,24 +318,24 @@ Quality Management Generic work products
...
 
     * - :need:`wp__platform_safety_manual`
-      - :need:`doc__score_platform_safety_manual`
-      - :ndf:`copy('status', need_id='doc__score_platform_safety_manual')`
+      - :need:`doc__platform_safety_manual`
+      - :ndf:`copy('status', need_id='doc__platform_safety_manual')`
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/quality/qms_report.rst:334]: need is comming from process_description

File: docs/quality/qms_report.rst [hunk 8]
```diff
@@ -314,24 +318,24 @@ Quality Management Generic work products
...
 
     * - :need:`wp__platform_safety_plan`
-      - :need:`doc__score_platform_safety_plan`
-      - :ndf:`copy('status', need_id='doc__score_platform_safety_plan')`
+      - :need:`doc__platform_safety_plan`
+      - :ndf:`copy('status', need_id='doc__platform_safety_plan')`
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/quality/qms_report.rst:338]: this need is comming for process_description

File: docs/index.rst [hunk 9]
```diff
@@ -147,44 +147,51 @@ Platform Architecture
...
-      :ref:`SW-platform Assumptions <platform_assumptions>`.
+      :ref:`Platform Assumptions <platform_assumptions>`.
+
+   .. grid-item-card::
+
+      :octicon:`graph;1.5em`
 
+      Platform Status
+      ^^^
+      Inform yourself about the status of the Platform ..... [tbd]
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/index.rst:158]: fine for the moment ... Status Page not available yet

File: docs/contribute/general/pullrequest.rst [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+  :alt: Detailed *S-CORE* Pull request workflow overview
+
+  Detailed *S-CORE* Pull request workflow overview
+
+Create a PR
+^^^^^^^^^^^
+
+The contributor (:need:`Contributor <rl__contributor>`) creates a PR.
+
+Reviewers ill be automatically assigned (:need:`Committer <rl__committer>`) based on the contributed content (ruleset as defined by the committers). In addition several checks for the contributed content (ToDo: Link to the description of the checks) will be started.
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/general/pullrequest.rst]: fixed with commit: fbeb944ff171941a5553603de4fc26727b0500be

File: docs/contribute/general/pullrequest.rst [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+   :security: NO
+   :realizes: wp__training_path[version==1]
+
+A Pull Request (**PR**) is the **ONLY** way to contribute **CODE** to the *S-CORE* project.
+
+The figure below shows a simplified workflow for a PR.
+
+* The contributor (:need:`Contributor <rl__contributor>`) starts by creating a PR:  `Creating a Pull Request (Github Docs) <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request>`_.
+* Required reviewers will be automatically assigned based on the contributed content (via CODEOWNERS).
+* If the content fullfils the review and acceptance criteria, a committer (:need:`Committer <rl__committer>`) will approve the *PR* and thus it can be merged.
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/general/pullrequest.rst]: fixed with commit: fbeb944ff171941a5553603de4fc26727b0500be

File: docs/contribute/index.rst [hunk 12]
```diff
@@ -12,65 +12,220 @@
...
 
-Additionally, two :ref:`steering committees <pmp_pm_steering_committees>`, the Technical Lead Circle and the Project Lead Circle, oversee the overall steering and planning of S-CORE.
+Additionally, :ref:`steering committees <pmp_pm_steering_committees>`, the Technical / Project Lead Circle, oversee the overall steering and planning of S-CORE.
 
 For further details on our project structure and planning, please refer to the :need:`Project Management Plan <doc__project_mgt_plan>`.
 
+How to Set Up your Environement
+===============================
+
+Find the necessary information for setting up your :ref:`Development Environement <setup_dev_environment>` ready.
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/index.rst]: fixed with commit: fbeb944ff171941a5553603de4fc26727b0500be

File: docs/contribute/development/cpp/cpp_code_analysis.rst [hunk 13]
```diff
@@ -27,9 +29,12 @@ Code Analysis C++
 
 Static Code Analysis
 ====================
-In order to fulfil the S-CORE related standard requirements a concept for *Static Code Analysis* needs to be established. Input for the analysis is based upon *MISRA* and *ISO26262* standards in accordance to the :need:`doc__cpp_coding_guidelines`.
+In order to fulfil the S-CORE related standard requirements a concept for *Static Code Analysis* needs to be established.
```
Comment (pahmann) [inline docs/contribute/development/cpp/cpp_code_analysis.rst]:
```suggestion
In order to fulfill the S-CORE related standard requirements *Static Code Analysis* is established.
```
Maybe we can write it already like this and not like "oh, and of course we also need to do this, so we will have a concept"

File: docs/contribute/development/cpp/cpp_code_analysis.rst [hunk 14]
```diff
@@ -27,9 +29,12 @@ Code Analysis C++
...
+Input for the analysis is based upon *MISRA* and *ISO26262* standards in accordance to the :need:`doc__cpp_coding_guidelines`.
 
-Checking those rules can partially be automated and implemented by a combination of different tools. Thus a mapping needs to be established which provides a linkage of all *MISRA* requirements to the respective tool requirements/rules. For *MISRA C++:2023* this mapping is established: :need:`here <doc__cpp_misra2023_rule_mapping>`
+Checking those rules can partially be automated and implemented by a combination of different tools.
+Thus a mapping needs to be established which provides a linkage of all *MISRA* requirements to the respective tool requirements/rules.
```
Comment (pahmann) [inline docs/contribute/development/cpp/cpp_code_analysis.rst]:
```suggestion
Thus a mapping is established which provides a linkage of all *MISRA* requirements to the respective tool requirements/rules.
```

File: docs/contribute/contribute_new_module/generate_documentation.rst [hunk 15]
```diff
@@ -12,12 +12,24 @@
    # SPDX-License-Identifier: Apache-2.0
    # *******************************************************************************
 
-Documentation generation
-==========================
+
+.. _generate_documentation:
+
+Gnereate Documentation
```
Comment (PandaeDo) [inline docs/contribute/contribute_new_module/generate_documentation.rst]:
```suggestion
Generate Documentation
```

**Review by RolandJentschETAS — DISMISSED** [review]

File: docs/contribute/contribute_new_module/generate_documentation.rst (hunk 15 above)
Comment (PandaeDo) [reply docs/contribute/contribute_new_module/generate_documentation.rst]: Don't like such changes. When I'm familiar with something I expect it in a location to click on it. With this the alphabetical order has changed so I have to find it again. Is this really needed?

File: docs/contribute/development/traceability_guidelines.rst [hunk 16]
```diff
@@ -12,29 +12,30 @@
...
+######################
+
+.. document:: Traceability Guide
+   :id: doc__traceability_guide
    :status: valid
    :version: 1
-   :safety: QM
-   :security: NO
-   :realizes: wp__requirements_proc_tool[version==1]
+   :safety: ASIL_B
```
Comment (PandaeDo) [inline docs/contribute/development/traceability_guidelines.rst:24]: What's the rational on this change?

File: docs/contribute/feature_request/feature_request.rst [hunk 17]
```diff
@@ -12,19 +12,18 @@
    # SPDX-License-Identifier: Apache-2.0
    # *******************************************************************************
 
+.. _feature_request_guideline:
 
-Feature & Enhancement Proposal (FEP)
-#####################################
+Feature Request Guideline
```
Comment (PandaeDo) [inline docs/contribute/feature_request/feature_request.rst:17]: It changes the headline but now it seems to me inconsistent to the following text.

File: docs/contribute/feature_request/feature_request.rst [hunk 18]
```diff
@@ -12,19 +12,18 @@
...
 .. document:: Feature Request Guideline
   :id: doc__feature_request_guideline
   :status: valid
   :version: 2
   :safety: QM
   :security: NO
-  :realizes: wp__training_path[version==1]
-
-.. _feature_request_guideline:
+  :realizes: wp__sw_development_plan, wp__training_path[version==1]
```
Comment (PandaeDo) [inline docs/contribute/feature_request/feature_request.rst:26]: Why it realizes the sdp?

File: docs/contribute/general/folder_structure_convention.rst [hunk 19]
```diff
@@ -12,15 +12,27 @@
...
+============================
+
+
+.. document:: Folder Structure Conventions
+   :id: doc__git_folder_structure_conventions
+   :status: valid
+   :version: 1
+   :safety: ASIL_B
+   :security: NO
+   :realizes: wp__sw_development_plan[version==1]
```
Comment (PandaeDo) [inline docs/contribute/general/folder_structure_convention.rst:27]: Also here I'm wondering why this will realize the sdp

File: docs/contribute/general/issue.rst [hunk 20]
```diff
@@ -0,0 +1,41 @@
...
+Issue Guide
+===========
+
+.. document:: Issue Guide
+   :id: doc__issue_guide
+   :status: valid
+   :version: 1
+   :safety: QM
+   :security: NO
+   :realizes: wp__training_path[version==1]
```
Comment (PandaeDo) [inline docs/contribute/general/issue.rst:26]: Please check for the complete proposal/PR. Seems to me that this need some input

**Review by masc2023 — DISMISSED** [review]

File: docs/introduction/integration_introduction.rst [hunk 21]
```diff
@@ -11,17 +11,26 @@
    #
    # SPDX-License-Identifier: Apache-2.0
    # *******************************************************************************
-.. _integration_process:
 
-Integration process
-====================
+.. _integration_introduction:
+
+Integration Introduction
```
Comment (PandaeDo) [inline docs/introduction/integration_introduction.rst:17]: Compare with the first sentence it seems to me that the renaming isn't helpful.

**Review by PandaeDo — CHANGES_REQUESTED** [review]
Please rework on it. I described some observations and stopped reading after a while. Looking forward for a new version to continue

File: docs/platform_management_plan/software_verification.rst [hunk 22]
```diff
@@ -141,7 +141,7 @@ There are the following different levels of integration and verification defined
 
 
   **Note:** These three levels translate to the levels of ISO 26262 part 6 clauses 9 to 11, where compliant testing with full coverage is tailored out for the embedded software.
-  The specific tailoring is described in the :need:`doc__score_platform_safety_plan`.
+  The specific tailoring is described in the :need:`doc__platform_safety_plan`.
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/platform_management_plan/software_verification.rst]: fixed with commit:

File: docs/platform_management_plan/safety_management.rst [hunk 23]
```diff
@@ -12,8 +12,8 @@
    # SPDX-License-Identifier: Apache-2.0
    # *******************************************************************************
 
-.. document:: Platform Safety Plan
-   :id: doc__score_platform_safety_plan
+.. document:: Safety Plan Platform
+   :id: doc__safety_plan_platform
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/platform_management_plan/safety_management.rst:16]: the change is correct

File: docs/contribute/index.rst [hunk 24]
```diff
@@ -12,65 +12,220 @@
...
+
+   development/cpp/cpp_code_analysis
+   development/cpp/cpp_coding_guidelines
+   development/cpp/cpp_misra_rules
+
+   development/fork/index
+
+
+   development/rust/rust_api_design
+   development/rust/rust_coding_guidelines
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/index.rst:217]: the folder had no contents and therefore deleted

File: docs/index.rst [hunk 25]
```diff
@@ -194,36 +201,35 @@ Reference Integration
 
    .. grid-item-card::
 
-      :octicon:`git-merge;1.5em`
+      :octicon:`iterations;1.5em`
 
-      Integration Baseline
+      Refernce Integration Process
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/index.rst]: done already

File: docs/index.rst [hunk 26]
```diff
@@ -67,53 +67,53 @@ Introduction
...
    .. grid-item-card::
-      :link: users_guide/whats_next/index
+      :link: users_guide/index
       :link-type: doc
       :text-align: center
 
       :octicon:`rocket;1.5em`
 
-      What's next?
+      User`s Guidline
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/index.rst]: fixed with commit:

File: docs/contribute/development/fork/index.rst [hunk 27]
```diff
@@ -0,0 +1,396 @@
...
+Recommendations
+
+1. Short-lived feature branches (e.g., ``topic`` or ```<username>/<topic>``)
+2. Open an individual PR for each change upstream
+3. Delete merged branches
+
+Notes
+
+- Your fork's ``main`` may either track upstream or remain unused.
+- Use pre-commit checks o detect internal-only patterns before pushing.
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/development/fork/index.rst]: fixed with commit:

File: docs/contribute/development/fork/index.rst [hunk 28]
```diff
@@ -0,0 +1,396 @@
...
+Example CI steps to generate a GitHub App token, configure git, and run Copybara:
+
+.. code::
+
+   - name: Generate GitHub App token
+     id: generate_token
+     uses: tibdex/github-app-token@v2
+     with:
+     app_id: ${{ secrets.GH_APP_ID }}
+     private_key: ${{ secrets.GH_APP_PRIVATE_KEY }}
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/development/fork/index.rst]: fix with commit:

File: docs/contribute/development/fork/index.rst [hunk 29]
```diff
@@ -0,0 +1,396 @@
...
+
+   - name: Configure Git
+     run: |
+     git config --global user.name  "Qorix Bot"
+     git config --global user.email "bot@qorix.dev"
+     echo "https://x-access-token:${{ steps.generate_token.outputs.token }}@github.com" > ~/.git-credentials
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/development/fork/index.rst]: fix with commit:

File: docs/contribute/development/fork/index.rst [hunk 30]
```diff
@@ -0,0 +1,396 @@
...
+
+   - name: Run Copybara
+     run: |
+     sed -i "s/{{BRANCH}}/${{ github.event.inputs.branch_name }}/g" copy.bara.sky
+     curl -LO https://github.com/qorix-group/copybara/releases/download/v20250508/copybara_deploy.jar
+     java -jar copybara_deploy.jar migrate copy.bara.sky publish_branch
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/development/fork/index.rst]: fixed with commit:

File: docs/contribute/development/fork/index.rst [hunk 31]
```diff
@@ -0,0 +1,396 @@
...
+
+
+3. How To work with forks
+=========================
+
+First and foremost see `GitHub's guide to working with forks <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo>`_
+
+Note that in enterprise environments forks will usually be created by forking into an company organization (e.g. ``my_company/score``) rather than a personal account. And those will be created by infrastructure administrators rather than individual developers.
+
+You can create such a fork e.g. via the GitHub CLI, `gh repo fork eclipse-score/score <(https://cli.github.com/manual/gh_repo_fork)>`_, or via an infrastructure-as-code process.
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/development/fork/index.rst]: fixed with commit:

File: docs/contribute/development/fork/index.rst [hunk 32]
```diff
@@ -0,0 +1,396 @@
...
+
+This guide helps companies (tool vendors, integrators, OEMs, suppliers) decide how to structure forks of S-CORE repositories to:
+
+- Contribute efficiently upstream
+- Integrate internal compliance & security workflows
+- Keep proprietary or distribution-specific assets separate
+- Avoid accidental leakage of secrets or internal IP
+- Automate promotion of reviewed code to the public community
+
+.. :note::
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/development/fork/index.rst]: fixed with commit:

File: docs/contribute/development/fork/index.rst [hunk 33]
```diff
@@ -0,0 +1,396 @@
...
+Depending on policy and compliance constraints, pick the simplest viable variant and document ownership for sync and publication tooling.
+
+2.1 Public-first Workflow
+-------------------------
+
+All S-CORE targeting contributions happen directly on the public fork.
+
+Recommendations
+
+1. Short-lived feature branches (e.g., ``topic`` or ```<username>/<topic>``)
```
[the remark this answers is not in the document]
Comment (anmittag) [reply docs/contribute/development/fork/index.rst]: fixed with commit:

File: docs/contribute/contribute_new_module/generate_documentation.rst (hunk 15 above)
Comment (anmittag) [reply docs/contribute/contribute_new_module/generate_documentation.rst]: typo fixed with commit: fbeb944ff171941a5553603de4fc26727b0500be

File: docs/contribute/feature_request/feature_request.rst (hunk 18 above)
Comment (anmittag) [reply docs/contribute/feature_request/feature_request.rst:26]: in general as @masc2023 already mentioned, we have to go through all documents whether they are training documents or sw development plan documents (or both). As this is some binding description on how in detail things have to be done, I assume it is a MUST criteria and therefore it goes into the dev plan.

File: docs/contribute/general/folder_structure_convention.rst (hunk 19 above)
Comment (anmittag) [reply docs/contribute/general/folder_structure_convention.rst:27]: as it is a convention = must => sw dev plan

File: docs/contribute/general/issue.rst (hunk 20 above)
Comment (anmittag) [reply docs/contribute/general/issue.rst:26]: in general as @masc2023 already mentioned, we have to go through all documents whether they are training documents or sw development plan documents (or both). As this is some binding description on how in detail things have to be done, I assume it is a MUST criteria and therefore it goes into the dev plan.

AND we have to add the doc-need to all files in score repos....

File: docs/introduction/integration_introduction.rst (hunk 21 above)
Comment (anmittag) [reply docs/introduction/integration_introduction.rst:17]: as a consequence as it is part of the Introduction chapter, I renamed it to integration_introduction...

File: docs/contribute/contribute_new_module/index.rst [hunk 34]
```diff
@@ -0,0 +1,49 @@
...
+
+- :ref:`Created a properly structured S-CORE module from scratch <create_new_score_module>`.
+- :ref:`Generated documentation from source and integrated it into the platform docs <generate_documentation>`.
+- :ref:`Build the module with Bazel and verified it compiles cleanly <build_module>`.
+- :ref:`Wired up CI/CD workflows so every change is automatically validated <cicd_workflows>`.
+- :ref:`Add the module into the reference integration to make it part of the platform <add_to_ref_integration>`.
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
```
Comment (RolandJentschETAS) [inline docs/contribute/contribute_new_module/index.rst:43]: Why hidden?

File: docs/score_releases/index.rst [hunk 35]
```diff
@@ -1,6 +1,6 @@
-.. _releases:
+.. _platform_releases:
 
-Releases and Roadmap
-====================
+Platform Releases
+=================
 
 Releases and roadmap are documented under the following link: `Reference Integration V1.0 <https://eclipse-score.github.io/reference_integration/main/s_core_v_1/>`_.
```
Comment (RolandJentschETAS) [inline docs/score_releases/index.rst:6]: roadmap -> roadmap for the next releases ?

**Review by RolandJentschETAS — APPROVED** [review]

File: docs/score_releases/index.rst (hunk 35 above)
Comment (anmittag) [reply docs/score_releases/index.rst:6]: copy / paste -not will be adapted later

File: docs/contribute/contribute_new_module/index.rst (hunk 34 above)
Comment (anmittag) [reply docs/contribute/contribute_new_module/index.rst:43]: because the documents are already visible with the refs above. avoiding double "double" referencing

File: docs/contribute/development/api_guideline.rst [hunk 36]
```diff
@@ -12,11 +12,13 @@
...
    # *******************************************************************************
 
+.. _api_guide:
+
 Designing API
 #############
 
-.. document:: API Guidelines
-   :id: doc__api_guidelines
+.. document:: API Guide
```
Comment (pahmann) [inline docs/contribute/development/api_guideline.rst]: The document is called guideline, why is the name now changed to "Guide". And where is the word "Guide" adding more value than having it named "Guideline" as it was before?

File: docs/contribute/development/setup_dev_environment.rst [hunk 37]
```diff
@@ -12,21 +12,23 @@
...
+.. document:: Development Environment Set Up
+   :id: doc__develop_environment_set_up
    :status: valid
    :version: 1
-   :safety: ASIL_B
-   :realizes: wp__sw_development_plan[version==1]
-   :security: YES
-
-.. _setting_up_dev_env:
+   :safety: QM
```
Comment (janis-qorix) [inline docs/contribute/development/setup_dev_environment.rst:24]: Why does the safety classification change from ASIL_B to QM? Could you clarify that?

File: docs/contribute/index.rst [hunk 38]
```diff
@@ -12,65 +12,220 @@
...
+
+
+.. document:: Contribution Guide
+   :id: doc__contribution_guide
    :status: valid
    :version: 1
-   :safety: ASIL_B
-   :security: YES
-   :realizes: wp__sw_development_plan[version==1]
+   :safety: QM
```
Comment (janis-qorix) [inline docs/contribute/index.rst:25]: Why does the safety classification change from ASIL_B to QM? Could you clarify that?

File: docs/introduction/useful_links.rst [hunk 39]
```diff
@@ -43,8 +54,8 @@ Project Documentation
 - `S-CORE Documentation <https://eclipse-score.github.io/score/main/>`_ — rendered project documentation (requirements, architecture, safety)
 - `Decision Records <https://eclipse-score.github.io/score/main/design_decisions/index.html>`_ — architectural and process decision records for the S-CORE project
 - `Platform Management Plan <https://eclipse-score.github.io/score/main/platform_management_plan/index.html>`_ — project governance and management plan
-- `Contribution Guidelines <https://eclipse-score.github.io/score/main/contribute/contribution_request/index.html#doc__contr_guideline>`_ — step-by-step guide for common contribution workflows
-- `Contribution Attribution <https://eclipse-score.github.io/score/main/contribute/general/contribution_attribution.html>`_ — how to properly attribute contributions
+- `Contribution Guidelines <https://eclipse-score.github.io/score/main/contribute/index.html>`_ — step-by-step guide for common contribution workflows
+- `Contribution Attribution <https://eclipse-score.github.io/score/main/contribute/general/contribution_attribution_guideline_guideline.html>`_ — how to properly attribute contributions
```
Comment (janis-qorix) [inline docs/introduction/useful_links.rst:58]: The link doesn't work

File: docs/introduction/useful_links.rst
Comment (janis-qorix) [inline docs/introduction/useful_links.rst]: The link doesn't work

Comment (janis-qorix) [inline docs/introduction/useful_links.rst]: The link doesn't work

Comment (janis-qorix) [inline docs/introduction/useful_links.rst]: The link doesn't work

**Review by janis-qorix — CHANGES_REQUESTED** [review]
Please add a rationale for the changed safety classification. 

Additional, please fix the broken link.
